.claude/agent-memory/codex-writer/MEMORY.md

```
@@ -0,0 +1,2 @@
+- [Codex companion invocation](codex_companion_invocation.md) — use `--cwd`/`--prompt-file` on codex-companion.mjs, `npm --prefix` for npm/tsc, to avoid chained-cd
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)